modules/bullet/rigid_body_bullet.cpp

```diff
@@ -866,7 +866,7 @@ void RigidBodyBullet::on_enter_area(AreaBullet *p_area) {
 
 	if (p_area->is_spOv_gravityPoint()) {
 		++countGravityPointSpaces;
-		assert(0 < countGravityPointSpaces);
+		ERR_FAIL_COND(countGravityPointSpaces < 0);
```

Comment:
If we want to match the previous code it should be `<= 0`. Or is allowing the `0` cases part of the fix?

Reply:
I think `< 0` is fine. Because of, that initialized with value of `0` at line `274`. Also it checked with `< 0` at line `379` as well. Why do you think that should be `<= 0`?

Reply:
Well I ask because the previous code was asserting on `countGravityPointSpaces > 0`, i.e failing on `countGravityPointSpaces <= 0`. I'm not saying the previous code was correct, just that this changes the logic so I wanted to make sure it's done on purpose.

Reply:
Oh, you are right. I missed that point. :)

Reply:
So, this is still changing the logic.
`assert` checks that a condition is true, and `ERR_FAIL_COND` checks that a condition is false, so if you want to change `assert` to `ERR_FAIL_COND`, you should negate the condition. The negation of `0 < countGravityPointSpaces` (i.e. `countGravityPointSpaces > 0` without the Java-ism) is `countGravityPointSpaces <= 0`.

Reply:
I'm always confused when I'm transforming between `assert` and `cond`. Thank you! Fixed.